merlin/utils.py

```diff
@@ -57,6 +57,27 @@
 DEFAULT_FLUX_VERSION = "0.13"
 
 
+def get_priority(api="rabbit", priority="low"):
+    apis = ["rabbit", "redis"]
```

Comment:
this should be replaced with a call to see what the current broker is. we do this in a few places, eg via

broker = CONFIG.broker.name.lower()